File: sdk/src/main/java/com/cnblogs/sdk/exception/CnblogsException.java
package com.cnblogs.sdk.exception;

/**
 * 博客园SDK异常
 * @author RAE
 * @date 2021/12/29
 * Copyright (c) https://github.com/raedev All rights reserved.
 */
public class CnblogsException extends RuntimeException {

    public CnblogsException() {
    }

    public CnblogsException(String message) {
        super(message);
    }

    public CnblogsException(String message, Throwable cause) {
        super(message, cause);
    }

    public CnblogsException(Throwable cause) {
        super(cause);
    }
}

File: sdk/src/main/kotlin/com/cnblogs/sdk/exception/CnblogsException.kt
package com.cnblogs.sdk.exception

/**
 * 博客园SDK异常
 * @author RAE
 * @date 2022/10/04
 * @copyright Copyright (c) https://github.com/raedev All rights reserved.
 */
class CnblogsException : RuntimeException {
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
}